tests/tests.py

```diff
@@ -1204,13 +1203,11 @@ def test_lets_encrypt_sanity(self):
             '-newkey', 'rsa:2048',
             '-subj', '/C=US/ST=Denial/L=Springfield/O=Dis/CN=www.example.com',
             '-passout', 'pass:foo',
-            '-keyout', '/tmp/key.key',
-            '-out', 'test_signed.crt',
+            '-keyout', os.path.join(gettempdir(), 'key.key'),
+            '-out', os.path.join(gettempdir(), 'test_signed.crt'),
```

Comment:
I think perhaps it's intended to write out to `signed.crt`, but the original code did not do that.